src/main/java/org/apache/commons/collections4/bag/UnmodifiableSortedBag.java

```diff
@@ -16,6 +16,11 @@
  */
 package org.apache.commons.collections4.bag;
 
+import org.apache.commons.collections4.SortedBag;
```

Comment:
Please don't change the import order in PRs, it makes it harder to review when there is noise compared to actual changes.